api/resolvers/growth.js

```diff
@@ -17,6 +17,7 @@ export function interval (when) {
 
 export function timeUnit (when) {
   switch (when) {
+    case 'custom':
     case 'week':
     case 'month':
       return 'day'
```

Comment:
So right here it should set the unit to `day`, not `hour`. But the charts always show x-axis labels like `9pm` or such, as if hours were selected. I do not understand exactly where the problem is.

Reply:
I'd guess that's something happening clientside in `components/charts.js > dateFormatter`